frontend/ui/data/css_tweaks.lua

```diff
@@ -303,12 +303,49 @@ h1, h2, h3, h4, h5, h6 { hyphens: none !important; }
                 separator = true,
             },
             {
-                id = "ruby_inline";
-                title = _("Render <ruby> content inline"),
-                description = _("Disable handling of <ruby> tags and render them inline."),
-                css = [[
-ruby { display: inline !important; }
-                ]],
+                title = _("Ruby"),
+                {
+                    title = _("About ruby"),
+                    info_text = _([[
+Ruby characters are small glosses on the Asian languages to show the logographs' (Hanzi, Kanji, Hanja, etc) pronunciation.
+These tweaks allow small stylistic changes and larger ones to make ruby easier to read or ignore.]]),
```

Comment:
Maybe something like this? "allow small stylistic changes and larger ones" is a bit inelegant, in any case.
```suggestion
These tweaks can help make ruby easier to read or ignore.]]),
```